source/quickstart.rst

```diff
@@ -6,22 +6,24 @@ Introduction
 
 Welcome to Manim DSL's documentation!
 
-Manim DSL is a Domain Specific Language that is made to help lecturers make animations for teaching data structures
-and algorithms
+Manim DSL is a domain specific language that helps educators create animations for teaching students about data structures
+and algorithms.
 
 Here is an example:
 
-.. code:: none
+.. code:: javascript
```

Comment:
Our code is not javascript. This will cause funny syntax highlighting and warnings when DSL does not look like javascript

Reply:
warnings in the docs generation

Reply:
We use it because its very similar to java script so we get some of the colors that way

Reply:
> warnings in the docs generation

I don't get any warnings

Reply:
Just like markdown as well for the README, if you want to add a syntax highlighter for sphinx for our language go ahead :laughing:

Reply:
But once it starts to diverge it'll throw warnings

Reply:
It's definitely not a long-term solution as once it starts diverging (which is probably quite soon) it might throw warnings. We can aim to make a syntax highlighting plugin for our language in a future iteration

Reply:
As I said, this will do for now, until somoene can be bothered to write the whole lexer for sphinx manually (which we aren't planning to do anytime soon) https://stackoverflow.com/a/16470058

Reply:
Also as far as I can tell it's not making any warnings for a file like this:

```
    let stack: Stack = new Stack;
    asdklj asdasd = asdas;
    stack.push(5);
    stack.pop();
```